templates/zerver/help/link-to-a-message-or-conversation.md

```diff
@@ -16,17 +16,20 @@ issue tracker.
 
 !!! warn ""
     This works for all views, including searches.
-    All URLs in Zulip are permalinks.
+    All URLs in Zulip are designed to be shareable.
 
 ## Link to a specific message
 
 {start_tabs}
 
 {!message-actions-menu.md!}
 
-1. Click **Link to conversation**.
+1. Click **Copy link to message**.
 
 {end_tabs}
 
-The copied link will take you to the topic of the relevant message, scrolled
-down to that message.
+This will copy to your clipboard a permanent link to the message,
+displayed in context (E.g. a stream + topic view).
```

Comment:
I think it would be clearer if this said something like: "displayed in its thread (i.e. topic view for messages in a stream).